### DISCUSSION TITLE
计算机专业学习路线 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#20] [Announcements]
# 计算机专业学习路线 | hackway.org

对计算机科学有追求的同学（读研、想进大厂或工作之余想提升自己），非常推荐你按照本学习路线花两三年的时间去深入学习，这对你今后的发展大有裨益；不过，如果你急于找工作（只有一年以内的时间），这种“科班式”的学习方式并不适合你，建议你按照“培训班式”的方式围绕企业招聘岗位去选择前端JavaScript、Java后端、Go等方向。

https://hackway.org/docs/cs/intro#comments

**Comment by Sandysomething** [comment]
紧跟大佬步伐，立志搞懂自然密码

**Reply by Gipsyfine** [reply]
微信群在哪呀 我可以加入嘛

**Reply by Rolling-meatballs** [reply]
同问
